index.html

```diff
@@ -4,14 +4,29 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend:wght@400;500&family=Manrope&family=Noto+Color+Emoji&display=swap" rel="stylesheet">
     <link rel = "stylesheet" href="style.css">
-    <script src = "class.js"></script>
+    <script src = "class.js" defer></script>
     <title>Awesome Books</title>
 </head>
 <body>
```

Comment:
- [ ] Kindly consider following the below-given design wireframe while designing your page to meet the project design requirements cause currently you have a totally different design
- 
<img width="900" alt="Screen Shot 2023-06-08 at 21 43 58" src="https://github.com/MasumaJaffery/Awesome-Books/assets/60146030/d277754a-13c9-411d-99ae-f5a27545e3c1">